server/index.js

```diff
@@ -10,8 +10,8 @@ app.use(express.static('client/dist'));
 
 
 
-app.listen(3000,()=>{
-  console.log('listening on port 3000');
+app.listen(4000,()=>{
```

Comment:
could you make sure to change this back to port 3000

Reply:
ok